nplbam/app/main.py

```diff
@@ -4,6 +4,10 @@
 
 
 @bp.route("/")
+@bp.route("/index")
 def index():
-    test_str: str = "Banana Bread"
-    return render_template("index.html", test=test_str)
+    return render_template("index.html")
+
+@bp.route("/query")
```

Comment:
Query should probably be in its own .py file (module). It will have its own calls relevant only to it (eg. /query_submit POST from the search form); so I think they should be separated as such.

Reply:
Yeah sure that seems fine